webui/react/src/components/kit/Dropdown.tsx

```diff
@@ -0,0 +1,75 @@
+import { Dropdown as AntDropdown } from 'antd';
+import { MenuProps } from 'antd/es/menu/menu';
+import { PropsWithChildren, useMemo } from 'react';
+import * as React from 'react';
+
+export interface MenuDivider {
+  type: 'divider';
+}
+
+export interface MenuOption {
+  className?: string;
+  danger?: boolean;
+  disabled?: boolean;
+  key: string;
+  label: React.ReactNode;
+  icon?: React.ReactNode;
```

Comment:
> Nothing too big, just a bunch of small things. I am curious why you decided not to tackle `src/components/Dropdown.tsx` here though, since it looks like it's only used in three places.

Regarding the 3 cases of that, I just wanted to run by design that they are ok with the change in visuals.

Reply:
Can you highlight those three cases and I'll look at them?

Reply:
there are 3 areas that use this old dropdown:
1. navigation when you click on profile
2. responsive filters
  a. trial detail workloads
  b. cluster historical usage filters
  c. trial chart (metric + filters?)
3. table action bar (current batch actions)

Did confirm quickly with Design that they are totally fine to replace (without arrows too)

Reply:
Added some more commits that migrate `components/Dropdown` over to the uikit version.